modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,217 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/creating-first-moa-cluster.adoc
+
+
+[id="moa-installing-prerequisites"]
+= Installing prerequisites
+
+Complete the following prerequisites before creating your {product-title} cluster.
+
+.Procedure
+
+. Select the AWS account you want to use.
++
+Unless you are just testing out MOA, it is recommended to use a dedicated AWS account to run any production clusters. If you are utilizing AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
++
+If you are using AWS organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
++
+As part of the cluster creation process, `moactl` will create an osdCcsAdmin IAM user. This user will leverage the IAM credentials you provide when configuring the AWS CLI.
++
+[NOTE]
+====
+This user will have Programmatic access enabled and the AdministratorAccess policy attached to it.
```

Comment:
@lamek what does it mean to have _Programmatic_ access? Maybe an AWS user innately understands this, I don't know.